### PR TITLE
Characterization (EDNA) related fixes

### PR DESCRIPTION
**Pull request by marcus-oscarsson**
Hi !

Several smaller fixes to make EDNA work properly, also important to be able to run certain workflows.

Marcus

### CODE REVIEW

File: mxcube3/ui/components/Tasks/Characterisation.js [hunk 1]
```diff
@@ -188,7 +190,7 @@ Characterisation = connect(state => {
   const subdir = selector(state, 'subdir');
   const prefix = selector(state, 'prefix');
   const runNumber = selector(state, 'run_number');
-  const fileSuffix = state.taskForm.fileSuffix === 'h5' ? '_master.h5' : '_????.cbf';
+  const fileSuffix = state.taskForm.fileSuffix === 'h5' ? '_master.h5' : 'cbf';
```
Comment (meguiraun) [inline mxcube3/ui/components/Tasks/Characterisation.js]: a point is missing (I know... a bit picky...)

Comment (marcus-oscarsson) [inline mxcube3/ui/components/Tasks/Characterisation.js]: Hey, actually there is not supposed to be a dot here, its just the extension without (.)

**Comment by meguiraun** [comment]
could you fix the conflict so I can accept?

**Comment by marcus-oscarsson** [comment]
Should be fine now

**Comment by meguiraun** [comment]
thanks!

**Pull request merged**